### ISSUE TITLE
Arrows between logit heatmaps (#4)

### ISSUE THREAD

**Issue by neelpawarcmu** [touretzkyds/BERT-insight#4]
We want "Z-shaped" arrows (number of arrows = number of answers) from start logits to end logits such that their heads point to the respective word from the heatmaps. Each arrow (number of arrows = number of answers) should have a number that matches the answer numbers from the model answers and should highlight when we hover over the corresponding answers. eg. hovering over answer 1 highlights arrow 1.

**Comment by neelpawarcmu** [comment]
Note that this might involve changing the data structure holding answers from a string to a list of answers, and the html `textarea` block to something like a canvas
